core/src/components/alert/test/a11y/alert.e2e.ts

```diff
@@ -0,0 +1,65 @@
+import AxeBuilder from '@axe-core/playwright';
+import { expect } from '@playwright/test';
+import type { E2EPage } from '@utils/test/playwright';
+import { configs, test } from '@utils/test/playwright';
+
+const testAria = async (
+  page: E2EPage,
+  buttonID: string,
+  expectedAriaLabelledBy: string | null,
+  expectedAriaDescribedBy: string | null
+) => {
+  const didPresent = await page.spyOnEvent('ionAlertDidPresent');
+  const button = page.locator(`#${buttonID}`);
+
+  await button.click();
+  await didPresent.next();
+
+  const alert = page.locator('ion-alert');
+
+  /**
+   * expect().toHaveAttribute() can't check for a null value, so grab and check
+   * the values manually instead.
+   */
+  const ariaLabelledBy = await alert.getAttribute('aria-labelledby');
+  const ariaDescribedBy = await alert.getAttribute('aria-describedby');
+
+  expect(ariaLabelledBy).toBe(expectedAriaLabelledBy);
+  expect(ariaDescribedBy).toBe(expectedAriaDescribedBy);
+};
+
+test.describe('alert: a11y', () => {
+  configs({ directions: ['ltr'] }).forEach(({ config, title }) => {
+    test(title('should not have accessibility violations when header and message are defined'), async ({ page }) => {
+      await page.goto(`/src/components/alert/test/a11y`, config);
```

Comment:
Can these `goto`s be moved to a `beforeEach`? Or is that not a pattern we're encouraging? Ditto for the other test files, or at least the ones that have more than one test.

Reply:
No, we can't use `beforeEach` here to house the `goto` calls. Placing `beforeEach` inside of the `configs` callback will cause multiple `beforeEach` blocks to be generated.

Reply:
Ok so I we moved the `configs` outside of the `test.describe` block we get a few benefits:

1. We can keep using `test.beforeEach` to do `page.goto`
2. We can eliminate the `title()` function usage on everything except for the `test.describe` block